pipeline/db/db.go

```diff
@@ -664,15 +665,13 @@ func GetAltApps(appID string) ([]AltApp, error) {
 	return result, nil
 }
 
-// precentifyArray produces a Postgres compatable 'like any' string intended
-// to be used as part of a larger query.
-func percentifyArray(arr *[]string) {
+func anyPatternFormat(arr *[]string) {
 	for i, v := range *arr {
 		(*arr)[i] = "%" + v + "%"
 	}
 }
 
-func percentifyStartifyArrayify(arr *[]string) {
+func startsWithPatternFormat(arr *[]string) {
```

Comment:
same as above. doesn't indicate what the function does. this name implies that it's checking something and returns a bool